server/test/bookingTest.js

```diff
@@ -1,161 +1,161 @@
-        done();
-      });
-  });
-});
-
-
-describe('POST User book without choosing trip', () => {
-  it('should return an error', (done) => {
-    chai.request(app)
-      .post('/api/v1/bookings')
-      // .send(booking[1])
-      .set('x-auth-token', token)
-      .set('Accept', 'application/json')
-      .end((err, res) => {
-        expect(res.body).to.be.an('object');
-        expect(res.body.status).to.equal(status.BAD_REQUEST);
-        expect(res.body.error).to.equal('"trip_id" is required');
-        expect(res.status).to.equal(status.BAD_REQUEST);
-        // expect(res.body.data.token).to.be.a('string');
-        done();
-      });
-  });
-});
-
-describe('DELETE User delete a booking', () => {
-  it('should delete a booking successfully', (done) => {
-    chai.request(app)
-      .delete('/api/v1/bookings/1')
-      .set('x-auth-token', token)
-      .set('Accept', 'application/json')
-      .end((err, res) => {
-        expect(res.body).to.be.an('object');
-        expect(res.body.status).to.equal(status.REQUEST_SUCCEDED);
-        expect(res.body.data.message).to.equal('Booking deleted successfully');
-        expect(res.status).to.equal(status.REQUEST_SUCCEDED);
-        // expect(res.body.data.token).to.be.a('string');
-        done();
-      });
-  });
-});
-
-describe('DELETE User delete a booking without access', () => {
-  it('should return an error', (done) => {
-    chai.request(app)
-      .delete('/api/v1/bookings/1')
-      .set('x-auth-token', NonUsertoken)
-      .set('Accept', 'application/json')
-      .end((err, res) => {
-        expect(res.body).to.be.an('object');
-        expect(res.body.status).to.equal(status.NOT_FOUND);
-        expect(res.body.error).to.equal('The User associated with this token doesn\'t exist.');
-        // expect(res.body.data.token).to.be.a('string');
-        done();
-      });
-  });
-});
-
-describe('GET User view all booking', () => {
-  it('should return all user bookings', (done) => {
-    chai.request(app)
-      .get('/api/v1/bookings')
-      .set('x-auth-token', token0)
-      .set('Accept', 'application/json')
-      .end((err, res) => {
-        expect(res.body).to.be.an('object');
-        expect(res.body.status).to.equal(status.REQUEST_SUCCEDED);
-        expect(res.status).to.equal(status.REQUEST_SUCCEDED);
-        // expect(res.body.data.token).to.be.a('string');
-        done();
-      });
-  });
-});
-
-describe('GET User view all booking', () => {
-  it('should return all user bookings', (done) => {
-    chai.request(app)
-      .get('/api/v1/bookings')
-      .set('x-auth-token', token)
-      .set('Accept', 'application/json')
-      .end((err, res) => {
-        expect(res.body).to.be.an('object');
-        expect(res.body.status).to.equal(status.REQUEST_SUCCEDED);
-        expect(res.status).to.equal(status.REQUEST_SUCCEDED);
-        // expect(res.body.data.token).to.be.a('string');
-        done();
-      });
-  });
-});
+// import jwt from 'jsonwebtoken';
```

Comment:
@victorkarangwa4 , Please delete all commented lines of code if they are not relevant. do not add redundant lines of codes to your PR